src/test/java/org/heigit/ohsome/ohsomeapi/controller/GetControllerTest.java

```diff
@@ -681,6 +681,30 @@ public void getUsersCountDensityGroupByBoundaryTest() {
         expectedValue * deltaPercentage);
   }
 
+  @Test
+  public void getUsersCountDensityByContributionTypeTest() {
```

Comment:
to make the naming slightly more similar to the following "contributions tests":

```suggestion
  public void getUsersCountDensityFilteredByContributionTypeTest() {
```